e2e/terraform/main.tf

```diff
@@ -132,7 +132,13 @@ go test -v ./e2e
 
 ssh into nodes with:
 ```
-ssh -i keys/${local.random_name}.pem ubuntu@${aws_instance.client_linux[0].public_ip}
+# server
+ssh -i keys/${local.random_name}.pem ubuntu@${aws_instance.server[0].public_ip}
+
+# clients
+%{ for ip in aws_instance.client_linux.*.public_ip ~}
+ssh -i keys/${local.random_name}.pem ubuntu@${ip}
+%{ endfor ~}
```

Comment:
This is not strictly necessary - but in some of my server specific tests, I'd like the ability of having no clients.